omnibus/Gemfile

```diff
@@ -5,4 +5,4 @@ gem 'omnibus', git: 'https://github.com/DataDog/omnibus-ruby.git', branch: 'data
 
 # Use Chef's software definitions. It is recommended that you write your own
 # software definitions, but you can clone/fork Chef's to get you started.
-gem 'omnibus-software', git: 'https://github.com/DataDog/omnibus-software.git', branch: 'master'
+gem 'omnibus-software', git: 'https://github.com/DataDog/omnibus-software.git', branch: 'jaime/ncursesgccfix'
```

Comment:
is this a branch we should merge into `omnibus-software/master`?

Reply:
Oh I forgot about this. This I needed to build locally.... I don't think it's an issue with the builders, but building will break for anyone on xenial without this (newer gcc's). If this doesn't hurt, I would probably merge that to `omnibus-software`.